proto/elys/leveragelp/types.proto

```diff
@@ -38,6 +38,7 @@ message Position {
     (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
     (gogoproto.nullable) = false
   ];
+  uint64 borrow_pool_id = 9;
```

Comment:
Since there is just the addition of a new field don't think we need to have Legacy Position, Protobuf allows that

Reply:
same, above method is working fine